lib/custom_commands/getPerformance.js

```diff
@@ -0,0 +1,45 @@
+var util = require("util");
+var EventEmitter = require("events").EventEmitter;
+
+
+function PerformanceMetrics() {
+    EventEmitter.call(this);
+}
+
+util.inherits(PerformanceMetrics, EventEmitter);
+
+PerformanceMetrics.prototype.command = function(fn, cb) {
+    this.cb = cb;
+    var self = this;
+    this.client.api
+        .execute(function() {
+                try {
+                    var performanceMetrics =[];
+                    var loadTime =  window.performance.timing.domContentLoadedEventEnd - window.performance.timing.fetchStart ;
+                    var fullPageLoad = window.performance.timing.loadEventEnd - window.performance.timing.fetchStart ;
```

Comment:
you'll probably want to do some kind of feature detection, since this is only a chrome thing right?  (or at least it would fail on old IE).